src/main/java/com/jcabi/dynamodb/maven/plugin/Instances.java

```diff
@@ -51,6 +51,9 @@
  * @since 0.1
  * @checkstyle ClassDataAbstractionCoupling (500 lines)
  * @see <a href="http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Tools.html">DynamoDB Local</a>
+ * @todo #44:30min Let's avoid code duplication between
```

Comment:
@simonjenga use in the puzzle same issue number.
a new issue will be created automatically with another id
the issue number in `@todo` needs for a trace purpose